### PR TITLE
[HWSKU] Define HWSKU Arista-7260CX3-Q64 and Arista-7260CX3-Q44

### PR DESCRIPTION
**Pull request by yxieca**
**- What I did**
Define HWSKU Arista-7260CX3-Q64 and Arista-7260CX3-Q44

Buffer configurations were copied from Arista-7260CX3-D108C8. It could use some improvement but as-is it should work.

### CODE REVIEW

File: device/arista/x86_64-arista_7260cx3_64/Arista-7260CX3-Q64/pg_profile_lookup.ini [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# PG lossless profiles.
+# speed cable size    xon  xoff   threshold xon_offset
+  50000  5m   1248    1248 56160  -3        2496
```
Comment (lguohan) [inline device/arista/x86_64-arista_7260cx3_64/Arista-7260CX3-Q64/pg_profile_lookup.ini]: need 40000 in the lookup table

Comment (yxieca) [reply device/arista/x86_64-arista_7260cx3_64/Arista-7260CX3-Q64/pg_profile_lookup.ini]: Thanks for catching this!

**Comment by yxieca** [comment]
Joe, I had to force push my branch to address the commit comment typo. Thanks for catching it.

**Comment by yxieca** [comment]
Made to 201811 branch on 2/14/2019

**Pull request merged**